compiler/plugins/input/TOSA/InputConversion/test/BUILD.bazel

```diff
@@ -17,6 +17,8 @@ iree_lit_test_suite(
     srcs = enforce_glob(
         [
             "apply_pdl_patterns_tosa.mlir",
+            # TODO(#17233)
+            # "apply_pdll_patterns_tosa.mlir",
```

Comment:
Huh, too bad.. would be good if we could run this tests... but I am guessing we cant "conditionally" add a test. So I am fine with this for now.